### DISCUSSION TITLE
New Github action details not showing

### DISCUSSION THREAD

**Discussion by batetolast1** [community/community#26430] [Actions]
<p>Hey to all! :slight_smile: I’ve just set up GitHub action with help of <a href="https://github.com/anmol098/waka-readme-stats" rel="noopener nofollow ugc">waka-readme-stats</a>, but I can’t access it’s details :frowning:</p>
<p><a href="https://github.com/batetolast1/batetolast1/actions" rel="noopener nofollow ugc">My profile repo actions</a> show new action, but when I click on the name, there are no details of this action and I can’t run it :confused:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/d/bd4ef54fbf4942065749447367736d8f013cd1c3.png" data-download-href="/uploads/short-url/r0HpmTlgMCyqBr0N0PGbCWymuJR.png?dl=1" title="123134023_621279441902155_8905858811513941378_n" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/4083634/181097626-aca54482-a694-488f-8f9c-8b129fe03b0a.png" alt="123134023_621279441902155_8905858811513941378_n" data-base62-sha1="r0HpmTlgMCyqBr0N0PGbCWymuJR" width="690" height="217" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd4ef54fbf4942065749447367736d8f013cd1c3_2_690x217.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd4ef54fbf4942065749447367736d8f013cd1c3_2_1035x325.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd4ef54fbf4942065749447367736d8f013cd1c3_2_1380x434.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd4ef54fbf4942065749447367736d8f013cd1c3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">123134023_621279441902155_8905858811513941378_n</span><span class="informations">1878×592 43.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can you offer some help here? :slight_smile:</p>

**Comment by Simran-B** [comment, marked as answer]
<p>Meanwhile there is:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/3/e3a3a5bd4c976e2c5d9573f5271255469c6d4323.png" data-download-href="/uploads/short-url/wtMZVhg3WJUeSU7dIjKFj16RmsH.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7819991/181097627-24ea1eaa-4983-4b7d-a437-b24e1c113860.png" alt="image" data-base62-sha1="wtMZVhg3WJUeSU7dIjKFj16RmsH" width="578" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/3/e3a3a5bd4c976e2c5d9573f5271255469c6d4323_2_578x500.png, https://aws1.discourse-cdn.com/github/original/3X/e/3/e3a3a5bd4c976e2c5d9573f5271255469c6d4323.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/3/e3a3a5bd4c976e2c5d9573f5271255469c6d4323.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/3/e3a3a5bd4c976e2c5d9573f5271255469c6d4323_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">689×596 21.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The configured events are:</p>
<pre><code class="lang-auto">on:
  workflow_dispatch:  
  schedule:
    - cron: "0 0 * * *"
</code></pre>
<p>which means the workflow will run daily at midnight, or if you trigger it manually:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/d/9d05e4302ac35bec9b5551216581ff4a79bb2db1.png" data-download-href="/uploads/short-url/mp5AgTELM6HUeuQ00ko6fFxWvq9.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7819991/181097628-5d845ab5-7f5e-4a25-ab42-2e0243af199e.png" alt="image" data-base62-sha1="mp5AgTELM6HUeuQ00ko6fFxWvq9" width="690" height="107" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/d/9d05e4302ac35bec9b5551216581ff4a79bb2db1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">934×145 7.26 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/batetolast1">@batetolast1</a>,</p>
<p>According to the screenshot you shared, it means the workflow has never been trigged to generate at least one workflow run, or the previous workflow runs had been deleted.</p>
<p>After the workflow was triggered, the workflow runs will be listed on the page.</p>

**Comment by batetolast1** [comment]
<p>Thanks guys for your help, it’s indeed working as expected :slight_smile:</p>
